swiss-ng-api.yaml

```diff
@@ -6279,42 +6281,72 @@ components:
         - debtorName
         - creditorAccount
         - creditorName
+        - requestedExecutionDate
```

Comment:
Isn't this in periodic payments? There should not be a `requestedExecutionDate` in periodic payment.